### PR TITLE
Unlocked Booze-O-Mat

### PR DESCRIPTION
**Pull request by rosieposieeee**
## About the PR

Adds a Booze-O-Mat variant that has no access requirements. Mainly for the perma one in Sub, and it would be useful for the maints bars where it has to be hacked *every* time.

NOTE: I hereby release all changes made in this PR under MIT license.

**Changelog**
:cl:
- add: Added a variant Booze-O-Mat that starts unlocked.

### CODE REVIEW

**Comment by IamVelcroboy** [comment]
This is probably a good idea. I wonder if we should reduce some of the inventory in it? Stick to the drinkware and basic mixers? Like, if you want the good stuff, go to the proper bar.

**Comment by rosieposieeee** [comment]
maybe, but eh, a bit out of scope for this

**Comment by IamVelcroboy** [comment]
> maybe, but eh, a bit out of scope for this

I don't think so. I think it's perfectly relevant. Just copy over an inventory for it and edit in a Delta folder. But I'm not saying it's mandatory. Just putting it out there for consideration.

**Comment by rosieposieeee** [comment]
fair, i really just meant this to be more of a bugfix though

File: Resources/Prototypes/DeltaV/Entities/Structures/Machines/vending_machines.yml [hunk 1]
```diff
@@ -48,3 +48,11 @@
       map: ["enum.WiresVisualLayers.MaintenancePanel"]
   - type: AccessReader
     access: [["Mail"]]
+
+- type: entity
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Structures/Machines/vending_machines.yml:52]: put in deltav namespace or add markers

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Entities/Structures/Machines/vending_machines.yml (hunk 1 above)
Comment (rosieposieeee) [reply Resources/Prototypes/DeltaV/Entities/Structures/Machines/vending_machines.yml:52]: its already in dv...? Resources/Prototypes/DeltaV/Entities/Structures/Machines/vending_machines.yml ??

Comment (deltanedas) [reply Resources/Prototypes/DeltaV/Entities/Structures/Machines/vending_machines.yml:52]: oh im blind

**Review by deltanedas — APPROVED** [review]

**Comment by Axionyxx** [comment]
Only came across this when redoing all changelogs that had failed, but non-player facing changes don't need changelog entries

**Pull request merged**